recipes/hdf5-lz4/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+PPATH="/usr/local/hdf5/lib/plugin"
+
+mkdir -vp ${PREFIX}/$PPATH
+
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+make -e HDF5_INSTALL="${PREFIX}/$PPATH"
```

Comment:
no worries. I wan't really even processing what you were suggesting, so copy/pasta fail